quiz.css

```diff
@@ -0,0 +1,64 @@
+article {
+	width: 30%;
+	float: left;
+	border: 1px solid lightblue;
+	margin-right: 1em; 
+	margin-top: 1.5em;
+	margin-bottom: .5em;
+}
+
+h1 {
+	border: 1px dotted lightgrey;
+	text-align: center;
+	margin-top: 0;
+	margin-bottom: 0;
+	padding-bottom: 1em;
+	padding-top: .7em;
+	display: block;
+	height: 2em;
+}
+
+.firstsec {
+	padding-top: .5em;
+}
+.secondsec {
+	padding: .5em .5em 0 .5em;
+	text-align: justify;
+}
+
+.thirdsec {
```

Comment:
Can you come up with a more semantic class name other than firstsec, secondsec, and thirdsec?